globals.go

```diff
@@ -134,8 +134,9 @@ const (
 
 	DefaultEvergreenConfig = ".evergreen.yml"
 
-	EvergreenHome = "EVGHOME"
-	MongodbUrl    = "MONGO_URL"
+	EvergreenHome   = "EVGHOME"
+	MongodbUrl      = "MONGO_URL"
+	MongodbAuthFile = "MONGO_DB_CREDS_FILE"
```

Comment:
extreme nit, but I think this should be either `MONGO_CREDS_FILE` or `MONGODB_CREDS_FILE`

Reply:
`MONGO_CREDS_FILE` it is